javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,129 @@
+let names = [];
+let todoList = [];
+//Main function of Voice Assistant
+function getReply(command) { 
+    if(command.startsWith('Hello')){
+        getResponse(command);
+    }else if(command.startsWith('What is my')){
+       getName(command);
+    }else if(command.startsWith('Add fishing')){
+        addToDoList(command);
+    }else if(command.startsWith('Add singing')){
+        updateToDoList(command);
+    }else if(command.startsWith('Remove')){
+        removeFromToDoList(command);
+    }else if(command.startsWith('What is on')){
+        getMyToDoList(command);
+    }else if(command.startsWith('What day')){
+        getTodayDate(command);
+    }else if(command.startsWith('What is')){
+        calculateSimpleMath(command);
+
+    }else if(command.startsWith('Set')){
+       console.log('Timer set for 4 minutes');
```

Comment:
This has to be dynamic as well. So if I put "Set a timer for 2 minutes" it could handle that as well.

Reply:
I changed it